docs/manual/introduction.tex

```diff
@@ -751,6 +751,10 @@
   \refqualclass{framework/qual}{AnnotatedFor} annotation, use unchecked code
   default annotations and suppress all type-checking warnings; see
   Section~\ref{compiling-libraries}.
+\item \<-AprioritizeStubs>
+  If both a stub file and a source file for a class are available,
```

Comment:
This isn't prioritizing stubs, it's merging the stub type with the source code type, so I find the flag name confusing.  It is also documented differently in `annotating-libraries.tex`. Or is this documentation wrong??

Reply:
The documentation was (mostly) correct. I fixed the code to make it actually correct, and renamed the flag accordingly.